### DISCUSSION TITLE
Installare HACS

### DISCUSSION THREAD

**Discussion by kisjeth** [indomus/forum#754] [HUB: Home Assistant]
Posto che ho seguito la guida (https://indomus.it/guide/installare-hacs-home-assistant-community-store-sul-proprio-hub/), non sono riuscito ad istallare hacs seguento la guida in questione perchè non sono stato capace di modificare il file yaml, ho provato come mi è stato consigliato tramite SSH, e la procedura sembrerebbe essere andata a buon fine (è presente la cartella HACS nel persorso home/homeassistant/.eccecc) ma quando vado ad aggiungere l'integrazione di HACS non mi porta a niente. Dopo un paio di minuti in cui dice di installare l'integrazione sparisce tutto e non me la fa configurare.

Home Assistant Core SENZA docker.

**Comment by snakuzzo** [comment]
Come già ti è stato detto in chat ieri e premesso che devi dare sempre un'occhiata ai log...

Loggati su RaspOS come utente applicativo
`sudo su -s /bin/bash homeassistant`

Spostati sulla config dir di HA
`cd /home/homeassistant/.homeassistant/config`

Lancia il seguente comando:
`wget -O - https://get.hacs.xyz | bash -`

Riavvia HA

Portati sotto `Configuration > Integrations` e clicca su `+ ADD INTEGRATION`

Seleziona HACS e prosegui con l'integrazione come da istruzioni a video

A questo punto dovresti aver terminato, se qualcosa non va devi guardare se logga qualcosa (e per qualcosa intendo qualcosa sempre riferito ad HACS)

**Reply by kisjeth** [reply]
Purtroppo dopo il tentativo di aggiornare python ad una versione superiore (fallito perché rimasto alla 3.7.3 iniziale) HA ha smesso di funzionare. A questo punto prendo in considerazione la strada di HASSIO o del doker riprendendo da capo tutto quanto.

**Reply by snakuzzo** [reply]
Da quello che ho letto in chat e qui ho capito che hai poca esperienza e praticità. Se posso darti un consiglio, per ora buttati su HassOS...poi in futuro quando avrai fatto pratica e avrai studiato un po' come funziona HA, valuterai altre opzioni.

**Reply by kisjeth** [reply]
Si, come già anticipato sono entrato in questo modo da 36 ore. Di Linux e simili sono completamente digiuno, quindi faccio un po' di fatica ad interfacciarmi.
Ho visto che in realtà posso installare HA anche sul NAS SYNOLOGY (cosa che ieri tra un'attesa e l'altra ho fatto con successo) ma non sono riuscito a capire come accedere da remoto. C'è qualche guida al riguardo per quanto riguarda SYNOLOGY?
Perché il Raspberry che sto usando è "in prova" e posso sempre renderlo indietro, se magari tramite Nas è più semplice (ed è già mio).

**Reply by snakuzzo** [reply]
Sul NAS solitamente hai due opzioni:

- VM HassOS -> [QUI](https://www.home-assistant.io/installation/alternative)
- Home Assistant Container (Core su Docker) -> [GUIDA](https://www.home-assistant.io/installation/alternative#synology-nas)

Visto che di Docker non ne saprai nulla ti consiglierei di importare una VM di HassOS
